security/oc_tls.c

```diff
@@ -1994,8 +2025,8 @@ oc_tls_add_new_peer(oc_tls_new_peer_params_t params)
   if ((params.endpoint->flags & TCP) == 0) {
     mbedtls_ssl_set_timer_cb(&peer->ssl_ctx, &peer->timer, ssl_set_timer,
                              ssl_get_timer);
-    oc_ri_add_timed_event_callback_seconds(
-      peer, oc_tls_inactive, (oc_clock_time_t)OC_DTLS_INACTIVITY_TIMEOUT);
+    oc_ri_add_timed_event_callback_ticks(peer, oc_dtls_inactive,
+                                         DTLS_INACTIVITY_TIMEOUT_TICKS);
```

Comment:
Maybe this timer could be used to close also TCP-TLS connection when we implement the closing idle sessions for TCP

Reply:
I'm checking the header for `mbedtls_ssl_set_timer_cb` and it should work for TLS as well. It is mandatory for DTLS, but not for TLS so that's probably the reason why it is like this.